### PR TITLE
Add matrix operation UI and scaffold add/subtract/multiply functions

### LINKED ISSUE #1: Add matrix input UI for calculator
**Is your feature request related to a problem? Please describe.**
There is currently no UI for the user to input a matrix which they would like to calculate against and it is a blank electron app.

**Describe the solution you'd like**
There should be two matrix inputs for each of the matrixes being calculated against and a dropdown for the operation which the user wishes to apply.

### PR DESCRIPTION
**Pull request by jribbink**
Closes #1 

 - Adds UI for matrix A, matrix B, and matrix C (result) (3x3 matrices only supported)
 - Adds dropdown for +/-/x operations
 - Automatically calculate result and assign to matrix C (using scaffolded helpers in `src/calculate.js`)
 - Adds unit test for Matrix.js

NEEDS README.MD UPDATES BEFORE MERGE

### CODE REVIEW

**Review by ItsMyFuneral — APPROVED** [review]
Looks good to me. UI displays two 3x3 matrix inputs and one 3x3 matrix output.

File: src/App.js [hunk 1]
```diff
@@ -27,15 +27,19 @@ function App() {
   useEffect(() => {
     let res;
     try {
+      const mapNumber = (m) => m.map((r) => r.map((x) => Number(x)));
```
Comment (jribbink) [inline src/App.js:35]: Inputs cannot be passed as string or else will be concatenated within calculate functions if not properly type-checked.  This safeguard prevents this.

Comment (jribbink) [reply src/App.js:35]: Wrapping inside of try/catch prevents invalid input from throwing error.

**Review by prokjs — APPROVED** [review]

**Comment by RileyComer** [comment]
Looks good to me!

**Review by RileyComer — APPROVED** [review]
Looks good!

**Pull request merged**